contributed_definitions/nyaml/NXmpes_liquid.yaml

```diff
@@ -5,7 +5,7 @@
 
 doc: This is the application definition for multidimensional photoelectron spectroscopy on liquids
 category: application 
-(NXmpes):
```

Comment:
as discussed with @flo also this was a copying over glitch it should be NXmpes_liquid